README.org

```diff
@@ -0,0 +1,35 @@
+* org.purefn/river
+  Minimalist Clojure wrapping of the Kafka Consumer API.
+
```

Comment:
This is insufficient. The doc should cover starting the local Kafka instance, starting up the processor, and what the expected result should be (e.g. generated events being in the define file).